uint16 id1
geometry_msgs/Pose2D pose1
uint16 id2
geometry_msgs/Pose2D pose2
uint16 id3
geometry_msgs/Pose2D pose3
uint16 id4
geometry_msgs/Pose2D pose4
